aips/aip-x.md

```diff
@@ -0,0 +1,237 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Permissioned Signer
+author: runtian-zhou
```

Comment:
Perhaps we could be inclusive on authorship, I think minimally beyond yourself of course, we could add Wolfgang, Igor, myself? Maybe Aaron?

Reply:
Added!